test/indent.txt

```diff
@@ -19,12 +19,74 @@ to go
 ++]
 end
 
+IfMultiBracket1
+to foo
+  let abc 0
+  if true [ set abc [
+    1 2 3 ]
+  ] [
+++  set abc [4 5 6]
+  ]
+end
+
+# I'm not certain this should be normative
+# This behavior wasn't specified prior to 6.0.1 and could probably change
+IfMultiBracket2
+to foo
+  let abc 0
+  if true [ set abc
+    [
+      1 2 3
+    ] ] [
+    set abc [4 5 6]
+  ]
+end
+
 IfPreindented
 to go
   if true
     [show 1]
 end
 
+IfPreindentedWithPriorLineMessedUp
+to go
+-  if true
+-    [show 1]
+end
+
+IfPreindentedMessedUpWithBlock
+to go
+-  if true
+-    [
+-      rt 90
+-      fd 1 ]
+end
+
+TopLineComment
+; comment at top
+; and second line
+to foo end
+
+FunctionAfterGlobals
+globals [
+++abc
+]
+to foo [def]
+-  do-something
+end
+
+## Again, this behavior wasn't specified prior to 6.0.1 and could probably be changed
```

Comment:
This looks like user code. It should probably be scrubbed and simplified with `foo`s and `bar`s. But again, it's weird enough that being normative about it feels odd.